cypress/e2e/article.cy.js

```diff
@@ -1,13 +1,38 @@
-describe('', () => {
-  before(() => {
+/* eslint-disable */
+/// <reference types='cypress' />
 
+describe('Article flow', () => {
+  let article;
+
+  beforeEach(() => {
+    cy.task('newArticle').then((newArticle) => {
+      article = newArticle;
+    });
+    cy.loginConduit();
   });
 
-  it('', () => {
+  it('should allow to create article', () => {
+    cy.get('.container > .nav > :nth-child(2) > .nav-link').click();
+    cy.findByPlaceholder('Article Title').type(article.title);
+    cy.get(':nth-child(2) > .form-control').type(article.description);
+    cy.findByPlaceholder('Write your article (in markdown)').type(article.body);
+    cy.findByPlaceholder('Enter tags').type(article.tag);
+    cy.get('.btn').click({ force: true });
 
+    cy.contains('h1', article.title).should('be.visible');
+    cy.url().should('include', 'article');
   });
 
-  it('', () => {
+  it('should allow to delete article', () => {
+    cy.get('.container > .nav > :nth-child(2) > .nav-link').click();
+    cy.findByPlaceholder('Article Title').type(article.title);
+    cy.get(':nth-child(2) > .form-control').type(article.description);
+    cy.findByPlaceholder('Write your article (in markdown)').type(article.body);
+    cy.findByPlaceholder('Enter tags').type(article.tag);
+    cy.get('.btn').click({ force: true });
 
+    cy.get(
+      '.article-actions > .article-meta > :nth-child(3) > .btn-outline-danger'
+    ).click();
```

Comment:
Try to choose the other selector, so we are sure you have clicked "Delete article" button

Reply:
Fixed